### PR TITLE
Some bikeshedding on the Notifications page

### PR DESCRIPTION
**Pull request by luckman212**
- added helptext to inform users that **multiple emails can be entered**
- reduce the use of "This is..."
- change "E-Mail" to "Email" or just "Mail" -- _more modern form_
- moved SSL/TLS option right below Port, where it logically belongs
- some other minor language & capitalization cleanups

### CODE REVIEW

File: src/usr/local/www/system_advanced_notifications.php [hunk 1]
```diff
@@ -230,93 +230,92 @@
...
 	'which notifications will be sent.');
 
 $section->addInput(new Form_Input(
 	'smtpport',
-	'SMTP Port of E-Mail server',
+	'SMTP port',
 	'number',
 	$pconfig['smtpport']
-))->setHelp('This is the port of the SMTP E-Mail server, typically 25, 587 '.
+))->setHelp('Port that the SMTP server accepts connections on. Typically 25, 587 '.
```
Comment (jim-p) [inline src/usr/local/www/system_advanced_notifications.php]: As long as we're changing this, put 587 first

File: src/usr/local/www/system_advanced_notifications.php [hunk 2]
```diff
@@ -230,93 +230,92 @@
...
-$group = new Form_Group('Secure SMTP Connection');
-$group->add(new Form_Checkbox(
+$section->addInput(new Form_Checkbox(
 	'smtpssl',
-	'Enable SSL/TLS',
+	'SSL/TLS encryption',
 	'Enable SMTP over SSL/TLS',
-	isset($pconfig['smtpssl'])
+	$pconfig['smtpssl']
 ));
```
Comment (jim-p) [inline src/usr/local/www/system_advanced_notifications.php]: We should probably add a note here that STARTTLS will only happen when this is **disabled**, and that it's different than this. Maybe:
> Enables direct SSL/TLS communication with the SMTP server, typically on port 465. When **disabled**, notifications will automatically attempt to use STARTTLS instead if supported by the SMTP server.

File: src/usr/local/www/system_advanced_notifications.php [hunk 3]
```diff
@@ -230,93 +230,92 @@
...
+))->setHelp('The email address that will appear in the From field.');
 
 $section->addInput(new Form_Input(
 	'smtpnotifyemailaddress',
-	'Notification E-Mail address',
+	'To email address',
 	'text',
 	$pconfig['smtpnotifyemailaddress']
-))->setHelp('Enter the e-mail address to send email notifications to.');
+))->setHelp('The email address to send notifications to. Multiple addresses may be '.
```
Comment (jim-p) [inline src/usr/local/www/system_advanced_notifications.php]: That first sentence is a bit awkward with the repeated 'to' there (it was already awkward in the original line). Maybe this instead:
> The email address to which notifications will be sent.

File: src/usr/local/www/system_advanced_notifications.php [hunk 4]
```diff
@@ -230,93 +230,92 @@
...
 $section->addInput(new Form_Button(
 	'test-smtp',
-	'Test SMTP Settings',
+	'Test SMTP settings',
 	null,
 	'fa-send'
-))->addClass('btn-info')->setHelp('A test notification will be sent even if the service is '.
-	'marked as disabled.  The last SAVED values will be used, not necessarily the values entered here.');
+))->addClass('btn-info')->setHelp('A test email will be sent even if the service is '.
+	'marked as disabled. If you have changed any settings, click SAVE before clicking '.
```
Comment (jim-p) [inline src/usr/local/www/system_advanced_notifications.php]: I think that "If you have changed..." sentence can be removed entirely, see 64f8ec62ddd57241e4a8a755cade7ca8cd9196a6

File: src/usr/local/www/system_advanced_notifications.php [hunk 5]
```diff
@@ -230,93 +230,92 @@
...
 
 $section->addInput(new Form_Checkbox(
 	'disablebeep',
-	'Startup/Shutdown Sound',
+	'Startup/Shutdown sound',
 	'Disable the startup/shutdown beep',
 	$pconfig['disablebeep']
-))->setHelp('When this is checked, startup and shutdown sounds will no longer '.
-	'play.');
+))->setHelp('When this is checked, startup and shutdown sounds will not be played.');
```
Comment (jim-p) [inline src/usr/local/www/system_advanced_notifications.php]: > When set, the firewall will not play startup and shutdown sounds

**Comment by jim-p** [comment]
In an effort to improve the tracking of changes and bug/feature requests, we have decided to require an entry on the [pfSense Redmine issue tracker](https://redmine.pfsense.org) associated with every pull request, and likewise the Redmine entry should also have a link back to the pull request.

If you could, please associate this PR to a Redmine issue either by locating an existing issue at https://redmine.pfsense.org or by creating a new issue. Add a link to the Redmine issue that points to this PR, and also add a link on the PR to the Redmine issue.

For more information, see https://doc.pfsense.org/index.php/Submitting_a_Pull_Request_via_Github

Thanks!

**Comment by sbeaver-netgate** [comment]
Please address the requested changes so that the PR can be merged. - Thanks.

**Comment by sbeaver-netgate** [comment]
Closing due to no activity in 3 months. Please resubmit if you can make the requested changes.

**Pull request closed without merging**